docs/gettingstarted/query_test_report.rst

```diff
@@ -45,6 +45,25 @@ You may find it useful to fetch all failed records and determine pass/fail crite
 
 .. _report_format_fields:
 
+Detailed Reports (``buildtest report --detailed``)
+---------------------------------------------------
+
+The **buildtest report --detailed** option can be used to show a detailed test report that may be of interest when
+examining a test. This option is synonymous to running ``buildtest report --format name,id,user,state,returncode,runtime,outfile,errfile,buildspec``.
+Shown below is an example output.
+
+.. dropdown:: ``buildtest report --detailed``
+
+    .. command-output:: buildtest report --detailed
+
+Note that ``--detailed`` and ``--format`` options can't be used together in ``buildtest report`` command they are
```

Comment:
Rewrite this whole section and put this is using a sphinx [note](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html).

State the following in the note

The ``detailed`` and ``--format`` options are mutually exclusive options because both options will alter the format fields when displaying test results.

Later in text after the note state the following

```
You will get the following error if you specify both options as shown below

dropdown:: ...
```

Reply:
fixed